Sources/Widgets/Widgets3D/EllipseWidget/state.js

```diff
@@ -25,11 +25,10 @@ export default function generateState() {
     })
     .addStateFromMixin({
       labels: ['ellipseHandle'],
-      mixins: ['origin', 'color', 'scale3', 'visible', 'direction'],
+      mixins: ['origin', 'color', 'scale3', 'visible', 'orientation'],
```

Comment:
isn't `corner` mixin missing ?

Reply:
Might be why it doesn't work :) Will look more, thanks